### DISCUSSION TITLE
Replace Preferences with JSON(b) fields #11451 ?

### DISCUSSION THREAD

**Discussion by almokhtarbr** [spree/spree#11657] [Q&A] [answered]
Hello any idea when it gonna be available the new preference implementation ? I'm gonna start new project and willing to use it but just noticed there is a new implementation of it! is there something i can help with ?

**Comment by damianlegawiec** [comment, marked as answer]
I think we will push this further into the future, so you can safely use Preferences, we've fixed them to work with Rails 7 on main branch so there's no blockers for this feature

**Comment by almokhtarbr** [comment]
Awesome Thanks!
